src/elements/content-sidebar/activity-feed/comment/Comment.js

```diff
@@ -132,19 +138,65 @@ class Comment extends React.Component<Props, State> {
         const { isConfirmingDelete, isEditing, isInputOpen } = this.state;
         const createdAtTimestamp = new Date(created_at).getTime();
         const createdByUser = created_by || PLACEHOLDER_USER;
+        const isTask = type === COMMENT_TYPE_TASK; // comment editing not supported
+        const { can_edit: canEdit = false, can_delete: canDelete = false } = permissions;
+        const isMenuVisible = (canDelete || (canEdit && isTask)) && !isPending;
 
         return (
             <div className="bcs-comment-container">
-                <div
+                <Media
                     className={classNames('bcs-comment', {
                         'bcs-is-pending': isPending || error,
                     })}
                 >
-                    {avatarRenderer(
-                        <Avatar className="bcs-comment-avatar" getAvatarUrl={getAvatarUrl} user={createdByUser} />,
-                    )}
-                    <div className="bcs-comment-content">
-                        <div className="bcs-comment-headline">
+                    <Media.Figure>
+                        {avatarRenderer(<Avatar getAvatarUrl={getAvatarUrl} user={createdByUser} />)}
+                    </Media.Figure>
+                    <Media.Body>
+                        {isMenuVisible && (
+                            <TetherComponent
+                                attachment="top right"
+                                className="bcs-comment-delete-confirm"
+                                constraints={[{ to: 'scrollParent', attachment: 'together' }]}
+                                targetAttachment="bottom right"
+                            >
+                                <Media.Menu isDisabled={isConfirmingDelete} data-testid="open-actions-menu">
+                                    {canEdit && isTask && (
+                                        <MenuItem
+                                            className="bcs-comment-menu-edit"
+                                            data-resin-target={ACTIVITY_TARGETS.INLINE_EDIT}
+                                            onClick={this.handleDeleteClick}
+                                        >
+                                            <IconPencil color={bdlGray80} />
+                                            <FormattedMessage
+                                                {...(isTask ? messages.taskEditMenuItem : commonMessages.editLabel)}
```

Comment:
The parent won't be displayed at all if `isTask` is false, right?

Reply:
true... the shortcircuit is a bit tacked-on

Reply:
Actually, I will pull up the `isTask` into the `canEdit` variable. Legacy tasks use inline edit so while that code is still here this should be supported.